src/main/java/org/springframework/data/domain/PageRequest.java

```diff
@@ -118,7 +119,7 @@ public Pageable next() {
 	 * @see org.springframework.data.domain.AbstractPageRequest#previous()
 	 */
 	@Override
-	public PageRequest previous() {
+	public Pageable previous() {
```

Comment:
We should rather change the other methods to return `PageRequest` as that is the more concrete type and we retain type-safety. Care to have a look?